include/ppx/imgui_impl.h

```diff
@@ -37,6 +37,7 @@ class ImGuiImpl
     virtual void   Shutdown(ppx::Application* pApp) = 0;
     virtual void   NewFrame();
     virtual void   Render(grfx::CommandBuffer* pCommandBuffer) = 0;
+    virtual void   ProcessEvent() {}
```

Comment:
this file belongs to a different PR?

Reply:
Yes, only as commented above, this is stacked on top of #388 only the last commit is for this PR.